File: JULIA/Cap06/c06ex01.jl
# c06ex01.jl

md = Array{Float64}(undef, 8)
global soma = 0.0

for i in 1:8
  print("Entre a media do aluno $i: ")
  md[i] = parse(Float64, readline())
  global soma += md[i]
end

media_geral = soma / 8
println("\nMedia geral: $(round(media_geral, digits=2))")

print("\nPressione <Enter> para encerrar... ")
readline()

File: CSHARP/Cap06/c06ex01.cs
// c06ex01.cs

using System;
using static System.Console;

class Program
{
    static void Main(string[] args)
    {
        double[] MD = new double[8];
        double SOMA = 0.0, MEDIA;

        for (int I = 0; I <= 7; I++)
        {
            Write("Entre a media do aluno {0}: ", I + 1);
            MD[I] = double.Parse(ReadLine());
            SOMA += MD[I];
        }

        MEDIA = SOMA / 8;

        WriteLine();
        WriteLine("Media geral: {0:F2}", MEDIA);

        Write("\nTecle <Enter> para encerrar... ");
        ReadLine();
    }
}
